addons/web/static/src/js/views/view_dialogs.js

```diff
@@ -314,6 +293,15 @@ var SelectCreateDialog = ViewDialog.extend({
             event.stopPropagation();
             this.$footer.find(".o_select_button").prop('disabled', !event.data.selection.length);
         },
+        field_changed: function (ev) {
+            ev.stopPropagation();
+        },
+        switch_view: function (ev) {
+            ev.stopPropagation();
+        },
+        do_action: function (ev) {
```

Comment:
I think all actions in kanban has been removed in selection mode.
If one day we really want to do an action in selection mode, we will be blocked by this override.
I think that it's not necessary anymore.

Reply:
@adr-odoo I think in which case we need action to perform in selection mode? I don't think so because action may lead to different view which we don't want to do in selection mode, I think that's why @ged-odoo told me to override all this events for ```SelectCreateDialog```